SU-HSDO/stanford_mrc
# MRCD8-209 inputs styles

# READY FOR REVIEW

# Summary
- Style inputs (specifically radio buttons and checkboxes)
- Added images for checkboxes

# Needed By (Date)
- End of sprint (2/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Check out one of the existing webforms (I suggest the form at /webform/vistior_funding_request

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-209

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
